assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -532,3 +532,6 @@ che.server.secure_exposer.jwtproxy.token.ttl=8800h
 che.server.secure_exposer.jwtproxy.auth.loader.path=/_app/loader.html
 che.server.secure_exposer.jwtproxy.image=eclipse/che-jwtproxy:latest
 che.server.secure_exposer.jwtproxy.memory_limit=128mb
+
+# Maximum size of the json processing pool
```

Comment:
It would be useful to add a few words that messages will be ignored if the pool is full. WDYT?